tests/test_epathlib.py

```diff
@@ -90,6 +88,31 @@ def test_glob(self):
         p = EPath("/tmp").glob("epathtestfile_[0-3].bin")
         assert len(list(p)) == 4
 
+    def test_s3_path_resolution(self):
```

Comment:
Is this test working? Earlier, I had to remove all tests that rely on S3 connectivity. See here: https://github.com/NVIDIA/Megatron-Energon/commit/1be5fb7881f65ebefb441763a3b337f66297b338

Reply:
Yes, it doesn't use S3 connectivity really, but is just checking the path resolution

Reply:
Ok. My plan was to mock the S3 connection to bring back the tests (when we have time).